test/integration/srv_resolver_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/letsencrypt/boulder/test"
 )
 
-type Config struct {
+type conf struct {
```

Comment:
This does resolve the issue, so LGTM, but it doesn't do so in a systematic way: a third file in //test/integration which wants its own config struct will have to select yet another unique name. It would probably be better to name this "srvResolverConfig" or something like that.

Reply:
I'll push another PR after this merges that differentiates it in both places. Just trying to ensure we get this release tagged.